conclave-host/build.gradle

```diff
@@ -11,6 +11,8 @@ dependencies {
     compileOnly "com.fasterxml.jackson.core:jackson-databind:$jackson_version"
 
     implementation "io.github.classgraph:classgraph:$classgraph_version"
+    implementation "org.apache.commons:commons-compress:$apache_commons_compress"
```

Comment:
We need to add this to the NOTICE file. Or use the `tar` command.

Reply:
Sorry, ignore my comment on using `tar` command. Just update the NOTICE file

Reply:
Done, have a look